keras_cv/layers/object_detection/multi_class_non_max_suppression_test.py

```diff
@@ -39,3 +41,60 @@ def test_decode_predictions_output_shapes(self):
         self.assertEqual(result["boxes"].shape, [8, 100, 4])
         self.assertEqual(result["classes"].shape, [8, 100])
         self.assertEqual(result["confidence"].shape, [8, 100])
+
+
+class XlaMlirBridgeNmsPredictionDecoderTest(tf.test.TestCase):
+    tf.config.experimental.enable_mlir_bridge()
```

Comment:
@bhack you can use pytest fixtures to accomplish this!

Call:
https://www.tensorflow.org/api_docs/python/tf/config/experimental/disable_mlir_bridge

after a `yield` statement in the fixture to disabe.

Reply:
Yes I've already tested that yesterday but the teardown it seems to not work correctly. 
I've pushed so you could check the code.